src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Compile.cs

```diff
@@ -38,7 +38,7 @@ public class Aapt2Compile : Aapt2 {
 		protected override int GetRequiredDaemonInstances ()
 		{
 			return Math.Min ((ResourcesToCompile ?? ResourceDirectories).Length, DaemonMaxInstanceCount);
```

Comment:
@dellis1972 : considering c626679763280937bac2dbd724d56f40fb11fbd8, shouldn't this have a check so that `GetRequiredDaemonInstances()` always returns at least 1?